### PR TITLE
HMAN-2 test PR

### PR DESCRIPTION
**Pull request by padmapriyanalam**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/HMAN-2

### Change description ###

Test PR

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: hmc [hunk 1]
```diff
@@ -192,6 +192,7 @@ function docker_build() {
     tagName=$(tag_name $project $branch)
     # hmi-api-gateway image from azure coming as hmi-api-gateway-web, to match and fix for local branch build
     imageName=${buildProject#"hmi-"}
+    cd $workspace && docker build . -t ccd/$imageName:$tagName || exit $?
```
Comment (danlysiak) [inline hmc]:
```suggestion
    (cd $workspace && docker build . -t hmc/$imageName:$tagName || exit $?)
```

File: hmc [hunk 2]
```diff
@@ -192,6 +192,7 @@ function docker_build() {
     tagName=$(tag_name $project $branch)
     # hmi-api-gateway image from azure coming as hmi-api-gateway-web, to match and fix for local branch build
```
Comment (RebeccaBaker) [inline hmc]: is this comment correct? as we don't have a gateway built yet?

File: hmc [hunk 3]
```diff
@@ -192,6 +192,7 @@ function docker_build() {
...
     imageName=${buildProject#"hmi-"}
```
Comment (danlysiak) [inline hmc]:
```suggestion
    imageName=${buildProject#"hmc-"}
```

File: hmc (hunk 2 above)
Comment (padmapriyanalam) [reply hmc]: Deleted the comment as it not relevant now

File: hmc (hunk 3 above)
Comment (padmapriyanalam) [reply hmc]: changed to hmc

**Review by danlysiak — DISMISSED** [review]

**Review by RebeccaBaker — APPROVED** [review]

**Review by danlysiak — APPROVED** [review]

**Pull request merged**